src/windows/functions.cpp

```diff
@@ -393,6 +393,38 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveEjectable(TCHAR driveLetter) {
+  TCHAR devicePath[8];
+  wsprintf(devicePath, TEXT("%c:\\"), driveLetter);
```

Comment:
Created an issue, so that we don't forget this: https://github.com/resin-io-modules/mountutils/issues/28